api/src/main/java/io/strimzi/api/kafka/model/JvmOptions.java

```diff
@@ -33,6 +33,7 @@
     private String xms;
     private Boolean server;
     private boolean gcLoggingEnabled = false;
+    private boolean tlsDebugEnabled = false;
```

Comment:
In a recent email discussion @tombentley pointed out that it can have more values than just `tls` which can help to reduce the output to only handshake for example. So I think this should be string and should be called `javaxNetDebug` maybe?

Reply:
Do we want also some enum of valid values?

Reply:
@tombentley WDYT? I think Strimzi might be ok, or?

Reply:
I wonder whether this is the right approach for enabling TLS debugging. All we're really doing here is setting a system property. It seem to me that sooner or later someone will come up with some other system property which needs to be set, and then we'll need a PR etc specifically for that. The alternative would be allowing a `Map<String, String> systemProperties` (or a `List<SystemProperty>` where `SystemProperty` is just a `name`, `value` pair) which would let the user set _any_ system properties they liked.

In any case, I think a String value is fine.

Reply:
I guess a map systemProperties would make sense as well. I'm fine with both.